subprojects/packagefiles/icu/source/io/meson.build

```diff
@@ -15,7 +15,10 @@ io_lib = library(
   install: true,
 )
 
+ioinc = include_directories('.')
+
 icuio_dep = declare_dependency(
+  include_directories: ioinc,
```

Comment:
```suggestion
icuio_dep = declare_dependency(
  include_directories: include_directories('.'),
```

Reply:
That's mostly a preference thing. I'd keep it as is.

Reply:
Unless it's used elsewhere, I don't see the point, especially with meson lack of scoping.

Reply:
https://github.com/mesonbuild/wrapdb/blob/a5cdf0b3285972bab78563d70dd77967a0d1c692/subprojects/packagefiles/icu/source/i18n/meson.build#L38-L44

I used i18n code as a reference.
Which is better?

Reply:
In the i18n case, it makes sense: `i18ninc` is used more than one time. (Although the inconsistent naming, not `i18n_inc`, irks me :P)

Reply:
Anyway, don't mind me, if @neheb is happy.

Reply:
icuio_depinc makes sense IMO

Reply:
Sorry I'm late.
I removed the variables and put them on one line.